### ISSUE TITLE
Game crashing and instant abandon penalty

### ISSUE THREAD

**Issue by LocalBirdFeeder** [ValveSoftware/Dota2-Gameplay#7213]
### Description

After accepting a game (tried it with Allpick, Turbo and Custom lobbies) my game will crash instantly.

**Comment by UtheMan** [comment]
Similar issue for me, crashing while loading demo game or while trying to watch the battle report.

**Comment by Swewww** [comment]
i changed vulkan to dx11 and all good,try it.

**Comment by dansamdevops** [comment]
> Similar issue for me, crashing while loading demo game or while trying to watch the battle report.
> 
> Once the demo game finishes loading, the game crashes. After I reach third tab in the battle report the game crashes as well. In both cases there is no error message or anything.

Yep exactly the same thing happens for me. The game crashes to desktop at the third part of the battle report. It also crashes during the banning phase of a new game. Fixed by switching from vulkan to dx11.

I'm using the latest AMD driver (23.2.2).
